### PR TITLE
Adjusted precompiles for individual claim

### PR DESCRIPTION
**Pull request by Dinonard**
**Pull Request Summary**

* Adjusted dapps staking precompiles for individual claim implementation

**TODO**
- [ ] change dependency to correct dapps staking pallet tag

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tests and/or benchmarks are included
- [ ] changed API client type definition or chain metadata

### CODE REVIEW

**Comment by hoonsubin** [comment]
Sorry! I didn't mean to remove the branch! I think I made a mistake when removing my local branch. Please reopen this!

**Review by Maar-io — APPROVED** [review]
LGTM

File: precompiles/dapps-staking/DappsStaking.sol [hunk 1]
```diff
@@ -8,46 +8,54 @@ interface DappsStaking {
...
-    /// @dev Withdraw all funds that have completed the unbonding process.
+    /// @notice Withdraw all funds that have completed the unbonding process.
     function withdraw_unbonded() external;
 
-    /// @dev Claim contract's rewards.
-    /// @dev Please use only for testing. This call will be replaced with 2 new calls in PR#6
+    /// @notice Claim contract's rewards for the specified era.
+    ///
+    /// @dev This call is deprecated and shouldn't be used anymore.
     function claim(address, uint128) external;
```
Comment (Maar-io) [inline precompiles/dapps-staking/DappsStaking.sol]: there are no projects in production yet. I'd remove this

**Review by Maar-io — APPROVED** [review]
LGTM

File: precompiles/dapps-staking/DappsStaking.sol (hunk 1 above)
Comment (Dinonard) [reply precompiles/dapps-staking/DappsStaking.sol]: Great, that makes it much cleaner.

**Pull request merged**